File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        unordered_map<int,int> mpp;
        for(int i=0;i<nums.size();i++){
            if(mpp.find(nums[i])!=mpp.end()) mpp[nums[i]]+=1;
            else mpp[nums[i]]=1;
        }
        for(auto it:mpp){
            if((it.second)>1) return true;
        }return false;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution:
    def containsDuplicate(self, nums):
        seen = set()  # Create an empty set
        for num in nums:
            if num in seen:  # If the number is already in the set, return True
                return True
            seen.add(num)  # Otherwise, add it to the set
        return False  # No duplicates found
